src/tools/sfdk/command.cpp

```diff
@@ -1674,7 +1674,7 @@ Worker::ExitStatus BuiltinWorker::runTools(const QStringList &arguments_, int *e
         if (!P::checkPositionalArgumentsCount(parser.positionalArguments(), 1, 1))
             return BadUsage;
 
-        const QString name = positionalArguments.first();
+        const QString name = parser.positionalArguments().first();
```

Comment:
How on earth did this end up in the commit :-o talking about the non-compiling code of course